agent-packaging/packaging-scripts/google-osconfig-agent.conf

```diff
@@ -0,0 +1,5 @@
+description "Google OSConfig Agent"
```

Comment:
same deal here, this is an existing file

Reply:
so, the packaging already had 2 version of these files, i actually did not add/remove anything from the packaging code. i just moved it from `packaging` directory to `agent-packaging/packaging-scripts`. I had a conversation with Andrew few days ago, and i remember we agreed on removing these duplicates.
anyhow, for now, i am reverting back everything to `packaging`.  i will send out a different PR to remove the duplicates.